### ISSUE TITLE
#7412 Voice-Over Problems with alt texts in Project-Overview & Initator-Dashboard 

### ISSUE THREAD

**Issue by jostertag** [liqd/a4-meinberlin#5174]
**URL:** https://meinberlin-dev.liqd.net/dashboard/organisations/permanent-test-projects/projects/ ; https://meinberlin-dev.liqd.net/projekte/
**user:** any
**expected behaviour:** The Voice-Over reads the alt text if a image can be seen (either new added alt text or the default alt text)
**behaviour:** 

1. The Voice-Over doesn't read the default alt text for the tile image in the project overview (see Screenshot 1). 
2. In case of the tile image, no distinction is made between alternative text and title information, but everything is read out one after the other at link level by the voice over. It would be better to have a subdivision or a note for the alternative text that it is an image (see Screenshot 2).
3. Voice-Over doesn't read the headerimage alt text in the initiator dashboard (just if a header-image was added with alt text and the header-image is therefore shown as the tile picture).

**important screensize:** any
**device & browser:** mac ff, safari & chrome (with voice over)
**Comment/Question:** @philli-m Regarding Point 3: I didn't know if it's on purpose that a header image serves automatically also as a tile image if no seperate tile image was uploaded by the initiators. If not, I can also create an issue here.

Screenshot?
**Screenshot 1:** 
here you can see for example that the voice reader doesn't read the default alt text, but reads the copyright information.
<img width="710" alt="Bildschirm­foto 2023-07-05 um 15 49 32" src="https://github.com/liqd/a4-meinberlin/assets/113608720/8d32ad74-1fa2-43cd-b22a-76ff8b88cd58">

**Screenshot 2:** 
here you can see that on link level the voice over reads out the alt text and the other information altogether. As a user with voice over you are not able to detect that the alt text refers to the image.
<img width="703" alt="Bildschirm­foto 2023-07-05 um 16 01 52" src="https://github.com/liqd/a4-meinberlin/assets/113608720/01ac8296-7611-487c-a96b-bd6666814e56">

**Comment by jostertag** [comment]
@philli-m Okay then I think you can leave point 2 out. I tested it as well with NVDA and there is a little pause between the image description and the reading of the title etc. 

And Point 1 & 3 are fixable, right?

**Comment by philli-m** [comment]
@jostertag yep other 2 points are fixable and for alt text I feel like adding 'image of ...' is good practice anyway
